provider/src/main/java/org/sunbird/keycloak/KeycloakSmsAuthenticator.java

```diff
@@ -0,0 +1,379 @@
+package org.sunbird.keycloak;
```

Comment:
@shriharshs I suggest adding test cases for this class. Please share a list of test cases which you believe will be relevant to test so we can prioritise accordingly.